### ISSUE TITLE
Add aria-labels

### ISSUE THREAD

**Issue by lassesteffen** [MakeItHappenJML/companyWebApp#5]
As a disabled person, I want to have the page readable for screen readers. Therefore elements should have aria-labels.

- [ ] read about it
- [ ] Implement if it makes sense
